<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tela de Vnedas</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>diamond-removebg-preview.png</normaloff>diamond-removebg-preview.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 175, 202);</string>
  </property>
  <widget class="QLineEdit" name="cod_vendas">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: #fff;
border: 1px solid #000;
border-radius: 8px;
}

QLineEdit:focus {
border: 2px solid #000000;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="total_vendas">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: #fff;
border: 1px solid #000;
border-radius: 8px;
}

QLineEdit:focus {
border: 2px solid #000000;

}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qtd_vendas">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: #fff;
border: 1px solid #000;
border-radius: 8px;
}

QLineEdit:focus {
border: 2px solid #000000;

}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CÓDIGO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>QUANTIDADE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>422</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TOTAL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>-10</y>
     <width>371</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../imds/logofundo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="sub_vendas">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>530</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: #fff;
border: 1px solid #000;
border-radius: 8px;
}

QLineEdit:focus {
border: 2px solid #000000;

}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>510</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>SUB TOTAL:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="itens_vendas">
   <property name="geometry">
    <rect>
     <x>642</x>
     <y>610</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: #fff;
border: 1px solid #000;
border-radius: 8px;
}

QLineEdit:focus {
border: 2px solid #000000;

}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>610</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Itens:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>521</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <column>
    <property name="text">
     <string>Codigo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nome</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantidade</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Descrição</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total </string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>580</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>diamond-removebg-preview.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pesquisa_vendas">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>245</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
	border-radius: 8px;

}

QPushButton::hover {

	color: rgb(255, 255, 255);
	border: 1px solid #ffffff;

}</string>
   </property>
   <property name="text">
    <string>Pesquisar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finalizar_vendas">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>590</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
	border-radius: 8px;

}

QPushButton::hover {

	color: rgb(255, 255, 255);
	border: 1px solid #ffffff;

}</string>
   </property>
   <property name="text">
    <string>FINALZAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="voltar_vendas">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>590</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
	border-radius: 8px;

}

QPushButton::hover {

	color: rgb(255, 255, 255);
	border: 1px solid #ffffff;

}</string>
   </property>
   <property name="text">
    <string>VOLTAR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
